fuel/schemes.py

```diff
@@ -18,6 +18,12 @@ class IterationScheme(object):
     sequential batches, shuffled batches, etc. for datasets that choose to
     support them.
 
+    Attributes
+    ----------
+    requests_examples : bool
```

Comment:
Hmm... it reads like a verb 'requests' and a noun 'examples' and is slightly confusing. 'produces_examples', 'requests_are_examples'?